File: exercise/kotlin/day14/src/main/kotlin/children/dtos/Child.kt
package children.dtos

import java.util.*

data class Child(
    var id: UUID,
    var firstName: String?,
    var middleName: String?,
    var lastName: String?,
    var birthCity: String?,
    var birthDate: String?,
    var gender: Gender,
    var address: Address?
)

enum class Gender {
    Girl,
    Boy
}

data class Address(
    var number: String,
    var street: String,
    var city: String,
    var countryId: Int
)

File: exercise/scala/day14/src/main/scala/children/dtos/Child.scala
package children.dtos

import java.util.UUID

case class Child(
                  var id: UUID,
                  var firstName: Option[String],
                  var middleName: Option[String],
                  var lastName: Option[String],
                  var birthCity: Option[String],
                  var birthDate: Option[String],
                  var gender: Gender,
                  var address: Option[Address]
                )

enum Gender {
  case Girl, Boy
}

case class Address(
                    var number: String,
                    var street: String,
                    var city: String,
                    var countryId: Int
                  )
